build/packaging/rpm/glideinwms.spec

```diff
@@ -1042,6 +1042,11 @@ rm -rf $RPM_BUILD_ROOT
 #%config(noreplace) %{_sysconfdir}/condor/scripts/frontend_condortoken
 
 %changelog
+* Thu Dec 1 2022 Marco Mambelli <marcom@fnal.gov> - 3.10.0-01.rc1
```

Comment:
should the release number be `3.10.0` instead of `3.10.0-01.rc1`? I did not see rc info in the previous releases below

Reply:
This line is modified when new RCs or the final release are made for the same release number